styles.css

```diff
@@ -0,0 +1,145 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+/* Media screen for mobiles with minimum width of 375px */
+@media screen and (min-width: 375px) {
+  body {
+    background-image: url(./images/background.png);
+    background-size: contain;
+  }
+
+  header {
+    padding: 20px;
+    display: flex;
+    align-items: center;
+  }
+
+  .img_class {
+    width: 76px;
+    height: 20px;
+  }
+
+  nav {
+    display: none;
+    flex: 1;
+    text-align: right;
+    font-family: Arial, Helvetica, sans-serif;
+    font-style: normal;
+    font-size: 1rem;
+    color: #344563;
+  }
+
+  nav ul {
+    display: inline-block;
+    list-style-type: none;
+  }
+
+  nav ul li {
+    display: inline-block;
+    margin-right: 20px;
+  }
+
+  a {
+    text-decoration: none;
+    color: #555;
+  }
+
+  .bigtext {
+    color: #091e42;
+    font-size: 3rem;
+    width: 327px;
+    height: 150px;
+  }
+
+  p {
+    color: #344563;
+    font-size: 16px;
+    width: 327px;
+    height: 146px;
+    line-height: 24px;
+    margin-top: 20px;
+  }
+
+  #introduction {
+    padding-left: 5%;
+    padding-top: 10%;
+  }
+
+  .connect {
+    color: #7f8cff;
+    font-size: 1.5rem;
```

Comment:
- [ ] Kindly consider changing this font size to `16px` for the same reasons I have described above in the heading and paragraph.